005_team_3_agent_design/005_team_3_agent_design.tex

```diff
@@ -192,6 +192,39 @@ \section{Results}
+
+\ref{fig:AvgDeaths} further shows how, even as the food in the platform changes, team 3’s agent always outperforms the selfish and random agents.
+
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=0.6\linewidth]{005_team_3_agent_design/images/Average_Deaths_of_Agents_Against_Total_Amount_of_Food_On_Platform.jpg}
+    \caption{Average deaths of agents against total amount of food on platform.}
+    \label{fig:AvgDeaths}
+\end{figure}
+
+In addition, we ran tests where our agent was placed into the tower along with another teams’ agent. Five agents from team 3 and five from the respective team in \ref{fig:AgainstOtherTeams} were simulated for 500 days and 100 total food on the platform. In \ref{fig:AgainstOtherTeams} you can see the cumulative deaths of both agent types.  
+
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=0.6\linewidth]{005_team_3_agent_design/images/Average_Deaths_When_Cooperating_With_Another_Agent_Type.jpg}
+    \caption{Average deaths when cooperating with another agent type.}
+    \label{fig:AgainstOtherTeams}
+\end{figure}
+
+This indicates that team 3 agents work best with agents from teams 5, 4, 2 and 7. Successful cooperation with multiple other agent types demonstrates our agent’s ability to adapt to different situations and work well with agents that are also capable of assuming a group identity, and creating a self-organising structure. This point is also illustrated by examining the large number of  total deaths in a tower with random agents. Their ignorant and  unpredictable actions means that the team 3 agents cannot adapt to their behaviour or form meaningful connections with them. Hence, the system cannot evolve, causing the deaths per time period to remain relatively constant, and the cumulative deaths to be high.
+
+Finally, we wanted to observe if the total number of agents in the tower had an effect on the outcome of the simulation, and whether fewer agents were able to communicate more effectively. This seemed to be the case, as proven by \ref{fig:DeathsNoAgents}. The lower the total number of agents, the more meaningful connections they can make and the stronger the social network. This is reflected in the two-agent tower data point, in which they successfully assume a group identity with no deaths. This occurs because both agents agree to leave each other enough food to survive. The greater the number of agents that are in the tower, the more difficult it is for them to form a self-preserving social network.In this case, the importance that the agents place on the collective identity does not outweigh their desire to form instinctive, self-preservative characteristics.  
```

Comment:
```suggestion
Finally, we wanted to observe if the total number of agents in the tower had an effect on the outcome of the simulation, and whether fewer agents were able to communicate more effectively. This seemed to be the case, as proven by \ref{fig:DeathsNoAgents}. The lower the total number of agents, the more meaningful connections they can make and the stronger the social network. This is reflected in the two-agent tower experiment, in which they successfully assumed a group identity with no deaths. This occurs because both agents agree to leave each other enough food to survive. The greater the number of agents that are in the tower, the harder it is for them to form a self-preserving social network.In this case, the importance that the agents place on the collective identity does not outweigh their desire to form instinctive, self-preservative characteristics.  
```